Ft login

Akamiz96 opened the pull request:
Login y Create an account. 
Falta trabajo en el diseño.

Akamiz96 committed:
Login and Create an account.

dhasane reviewed (changes requested):
aun no sirve para hacer login, ya prove el funcionamiento de creacion de usuario, pero toca acceder directamente desde la url, seria util que funcionaran los links

Akamiz96 committed:
Creation of home page

Creation of home page

Akamiz96 committed:
Home page done

Home page done with routes to Login and Registrarse.

Akamiz96 committed:
Diseño home, login, registro con route

Akamiz96 committed:
Error message. First edition

Akamiz96 committed:
fix visibility of error message

Akamiz96 committed:
login and register working with name

Akamiz96 committed:
Remove side menu

Akamiz96 committed:
Button home

Akamiz96 committed:
Merge branch 'develop' into ft-login

Akamiz96 commented:
Ya listo el menu con el registro y login

Akamiz96 committed:
Change in auth menu

Akamiz96 committed:
Post user data to server

Faltan pedir algunos datos del perfil de usuario pero ya hace post al server LOCALHOST

Akamiz96 committed:
Save user data in database

Akamiz96 committed:
Merge branch 'develop' into ft-login
